wire/cli/src/main.rs

```diff
@@ -49,7 +60,72 @@ async fn main() -> Result<(), anyhow::Error> {
             no_keys,
             always_build_local,
         } => {
+            let stdin = std::io::stdin();
+
+            // set fd to nonblocking so it won't hang the program in case if
+            // there is no piping to stdin
+            let fdflags = fcntl(stdin.as_raw_fd(), F_GETFL)?;
+            let newflags = OFlag::from_bits_retain(fdflags | O_NONBLOCK);
+            fcntl(stdin.as_raw_fd(), F_SETFL(newflags))?;
+
+            let mut handle = stdin.lock();
+
+            // This could be improved probably, but for now this stays.
+            let nodes = {
+                let mut buf: Vec<u8> = Vec::new();
+                let mut c: u8 = 0;
+                loop {
+                    match handle.read_until(b'\n', &mut buf) {
+                        Result::Ok(0) => break,
+                        Result::Ok(_) => {
+                            break;
+                        }
+                        Err(e) => match e.raw_os_error() {
+                            Some(11) => {
+                                // Resource temporarily unavailable is a normal
+                                // error response when reading from stdin
+                                // nonblocking
+                                //
+                                // as for if there should be a counter for this,
+                                // this is undecided.
+                                if c >= 2 {
+                                    break;
+                                }
+                                c += 1;
+                            }
+                            Some(code) => {
+                                bail!("unhandled stdin error: {code} {e}");
+                            }
+                            None => bail!(e),
+                        },
+                    }
+                }
+
+                String::from_utf8(buf)
+                    .unwrap()
+                    .trim_end()
+                    .split(',')
```

Comment:
Many cli tools split their stdin by newline, I think splitting by whitespace is also appropriate. I think this should be split by `,`, ` `, and `\n`

Reply:
implemented by: https://github.com/wires-org/wire/pull/140/commits/cc9982932a2d144be434840e4f677054da5df63e